### PR TITLE
Test plan for next hop split

### PR DESCRIPTION
**Pull request by TACappleman**
What is the motivation for this PR?
This is the test plan for the next hop group split enhancement, as covered in PR 712 (https://github.com/Azure/SONiC/pull/712)

### Description of PR
Added test plan for next hop group split enhancement (https://github.com/Azure/SONiC/pull/712)

### Type of change
- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [X ] Test case(new/improvement)

#### What is the motivation for this PR?
Present test plan for enhancement before writing the tests.

#### How did you do it?
N/A

#### How did you verify/test it?
N/A

#### Any platform specific information?
N/A

#### Supported testbed topology if it's a new test case?
N/A

### Documentation 
Commit is documentation

### CODE REVIEW

File: docs/testplan/Next-hop-split-test-plan.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* Update the route to reference the new next hop group
+
+### Pass/Fail Criteria
+*  After the route is created verify that traffic for the prefix is routed via the next hop
+*  After the route is updated verify that traffic for the prefix is routed via a next hop in the group
+
+## Test Case 2. IPv6 routes
+
+### Test Objective
+Verify that IPv4 routes and next hop groups can be programmed and cause the correct data plane behaviour.
```
Comment (qiluo-msft) [inline docs/testplan/Next-hop-split-test-plan.md:46]: >IPv4 [](http://example.com/codeflow?start=12&length=4)

IPv6

**Comment by yxieca** [comment]
@TACappleman there is a minor type in the HLD, can you fix that so that we can get the PR merged?

File: docs/testplan/Next-hop-split-test-plan.md (hunk 1 above)
Comment (TACappleman) [reply docs/testplan/Next-hop-split-test-plan.md]: Fixed

**Comment by TACappleman** [comment]
@yxieca Thanks, fixed

**Review by qiluo-msft — APPROVED** [review]
LGTM. Please check with other reviewers.

**Review by yxieca — APPROVED** [review]

**Pull request merged**